core/domain/exp_domain_test.py

```diff
@@ -534,6 +535,11 @@ def test_validation(self):
         self._assert_validation_error(
             exploration, 'Expected outcome param_changes to be a list')
 
+        outcome.param_changes = [param_domain.ParamChange(
+            0, 'generator_id', {})]
+        self._assert_validation_error(
+            exploration, 'Expected param_change name to be a string')
```

Comment:
Should we make it `Expected param_change name to be a string, received <something>`?

Reply:
done